FileIO/TetGenInterface.cpp

```diff
@@ -230,8 +231,18 @@ MeshLib::Mesh* TetGenInterface::readTetGenMesh (std::string const& nodes_fname,
 		return nullptr;
 	}
 
+	MeshLib::Properties properties;
+	// remove material vector again if all mat-ids are "0"
+	if (*(std::max_element(materials.cbegin(), materials.cend())) > 0)
```

Comment:
Better to use `any_of()` algorithm here both in readability and speed (it will stop looking after first non-zero element is found).